metric/go.mod

```diff
@@ -0,0 +1,54 @@
+module go.opentelemetry.io/otel/metric
+
+go 1.16
```

Comment:
This should probably be `1.14` or whatever our minimum supported version is.

Reply:
Fixed in latest set of commits.